lib/slack-ruby-bot-server/api/endpoints/teams_endpoint.rb

```diff
@@ -84,7 +84,7 @@ class TeamsEndpoint < Grape::API
             if team
               team.ping_if_active!
 
-              team.update_attributes!(
+              team.update!(
```

Comment:
Is this needed?

Reply:
This is the only *substantive* change of this PR. update_attribute! has been an alias to update! since rails 4.0 and was finally deprecated in rails 6.1 and dropped in rails 7.0

Reply:
Got it!